tests/import_test.go

```diff
@@ -325,7 +325,8 @@ var _ = Describe("[Istio] Namespace sidecar injection", func() {
 		By(fmt.Sprintf("Create new datavolume %s", dataVolume.Name))
 		// We set the Immediate Binding annotation to true, to eliminate creation of the consumer pod, which will also fail due to the Istio sidecar.
 		dataVolume.Annotations[controller.AnnImmediateBinding] = "true"
-		dataVolume.Annotations[controller.AnnPodSidecarInjection] = "true"
+		dataVolume.Annotations[controller.AnnPodSidecarInjectionIstio] = "true"
+		dataVolume.Annotations[controller.AnnPodSidecarInjectionLinkerd] = "true"
```

Comment:
Wondering if it would make sense to have a separate permutation of the test for Linkerd.
Likely not - because only one of these would be present on a cluster, right?
If so, we could add a comment about this

Reply:
Either would work yes, but you're right - only one of these meshes _should_ be present on a cluster. Nothing would stop you from deploying both meshes but reckon you'll get some unexpected behaviour. Generally its one or the other. If think its necessary I can add a comment on this?